src/tpl/components/navigation.nunj

```diff
@@ -1,18 +1,18 @@
-<div class="navigation">
+<div class="navigation js-navigation">
     <div class="navigation__bar navigation__main">
         <div class="navigation__container">
-            <a href="#" class="button navigation__button main__button">
+            <a id="js-index" href="#" class="button navigation__button main__button">
```

Comment:
use vicee versa  appriach. Toggler has data-*, content has id. You keep in a toggler a reference to the content.